scss/bitstyles/organisms/sidebar/Sidebar.js

```diff
@@ -47,9 +44,12 @@ const labels = [
   'Sales',
 ];
 
-labels.forEach((label) =>
-  buttonList.appendChild(listItem.appendChild(buttomComponent(label)))
```

Comment:
This cause `appendChild` returns the child, not the parent, so this code was resulting in buttons being appended directly to the list, and the list item was unused.